### PR TITLE
Add Sentry crash logging and global error boundary

### PR DESCRIPTION
**Pull request by the-kenneth**

### CODE REVIEW

File: src/core/ErrorBoundary.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  error: Error;
+  componentStack: string | null;
+  eventId: string | null;
+  resetError(): void;
+}) {
+  return (
+    <View style={[styles.outerContainer]}>
+      <View style={[styles.innerContainer]}>
+        <Image source={covidIcon} style={styles.logo} />
+        <Header3Text>Oops! Something went wrong.</Header3Text>
```
Comment (selvach) [inline src/core/ErrorBoundary.tsx:22]: Could you add to i18n strings, please 🙏

Comment (the-kenneth) [reply src/core/ErrorBoundary.tsx]: Updated

File: assets/lang/sv-SE.json [hunk 2]
```diff
@@ -1314,5 +1314,11 @@
   "generic": {
     "share-this-with-someone": "Share this with someone",
     "content-can-not-be-loaded-atm": ""
+  },
+
+  "error-boundary": {
+    "something-went-wrong": "Hoppsan! Något gick fel.",
```
Comment (bionara) [inline assets/lang/sv-SE.json:1320]: Hoppsan is a great word!

File: ios/Podfile.lock [hunk 3]
```diff
@@ -668,4 +686,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 90db9c8877350d67cdc023e86d851d41d5b064bd
 
-COCOAPODS: 1.9.3
```
Comment (bionara) [inline ios/Podfile.lock:671]: i this downgrade intentional? Would probably be best if we as a time agreed on a version (1.10.0 seems good!)

**Review by bionara — DISMISSED** [review]
Look good, nice one! Left a comment about cocoapods version downgrade

File: ios/Podfile.lock (hunk 3 above)
Comment (the-kenneth) [reply ios/Podfile.lock]: Nice catch, was not intentional

File: assets/lang/en.json [hunk 4]
```diff
@@ -1316,5 +1316,11 @@
   "generic": {
     "share-this-with-someone": "Share this with someone",
     "content-can-not-be-loaded-atm": "This content can not be loaded at the moment."
+  },
+
+  "error-boundary": {
+    "something-went-wrong": "Oops! Something wet wrong.",
```
Comment (selvach) [inline assets/lang/en.json:1322]: wet -> went

File: assets/lang/en.json [hunk 5]
```diff
@@ -1316,5 +1316,11 @@
...
+    "encountered-error": "Sorry but the app has encountered an error:",
+    "notified": "The development team has been notified and should hopefully fix this issue asap! Try to restart the app to continue."
```
Comment (selvach) [inline assets/lang/en.json:1324]: Was this copy from Mathias & Christina?

File: assets/lang/en.json [hunk 6]
```diff
@@ -1319,5 +1319,11 @@
   "generic": {
     "share-this-with-someone": "Share this with someone",
     "content-can-not-be-loaded-atm": "This content can not be loaded at the moment."
+  },
+
+  "error-boundary": {
+    "something-went-wrong": "Oops! Something wet wrong.",
```
Comment (bionara) [inline assets/lang/en.json:1325]: typo - should be went not wet

**Review by bionara — APPROVED** [review]
Code looks good!

**Review by selvach — APPROVED** [review]

**Pull request merged**